notebooks/langchain-pipeline-rosa.ipynb

```diff
@@ -0,0 +1,894 @@
+{
```

Comment:
<p>Are we planning to compare the others in the coming notebooks? If so we can indicate that here and if not we could edit the model dictionary. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>A model comparison is certainly out of scope for this notebook - but I believe it is still useful to be able to pick among a list when running this notebook locally, to experiment with different models.</p><p> </p><p>I am adding a couple of sentences to explain this.</p>